File: examples/equos-python-integration/src/python_integration/create_brain.py
from equos.models import CreateEquosBrainRequest

from python_integration.client import get_client


def create_brain() -> None:
    client = get_client()

    brain = client.brains.create(
        CreateEquosBrainRequest(
            instructions="You are a helpful assistant that translates English to French.",
            greeting_message="Bonjour! What would you like me to translate?",
        )
    )
    print("Brain created:", brain.id)


if __name__ == "__main__":
    create_brain()

File: examples/equos-nodejs-integration/src/create-brain.ts
import { client } from "./client.js";

const brain = await client.brains.createBrain({
  createEquosBrainRequest: {
    instructions:
      "You are a helpful assistant that translates English to French.",
    greetingMessage: "Bonjour! What would you like me to translate?",
  },
});

console.log("Brain created:", brain.id);
console.log(brain);
